assets/src/scripts/monitoring-location/components/hydrograph/download-links.js

```diff
@@ -53,19 +21,22 @@ const createUrlForDownloadLinks = function(currentIVDateRange, queryInformation,
 * @param {store} store - The Redux store, in the form of a JavaScript object
 * @param {String} siteno- a USGS numerical identifier for a specific monitoring location
 */
-/*
 export const renderDownloadLinks = function(elem, store, siteno) {
+    const monitoringLocations = [siteno]; // The method that processes the service URL expects an array of locations
+
     elem.call(link(store, (elem, {
-        currentIVDateRange,
-        parameterCode,
-        showIVTimeSeries,
-        queryInformation,
-        medianData,
-        anyVisibleGroundwaterLevels,
-        requestTimeRange
+        currentTimeRange,
+        priorYearTimeRange,
+        inputs,
```

Comment:
The time parameters part of this object aren't used. I wonder if it would be clearer to pass in the parameterCode, showCompareIV and showMedian directly. I was confused that you were grabbing both the inputs and the time ranges (I understand why you need the time ranges though). I don't feel real strongly about this so I'm fine with this staying as is.

Reply:
Actually thinking about this, you could use the time portion of inputs directly to get the url for primary. You would still need to get the priorYearTimeRange to get the right time range for the compare IV data though. But these are only suggestions. There are obviously pros and cons both ways.

Reply:
I went back and forth on this one. I actually changed the code and then changed it again, then slept on it to see if I felt the same way in the morning :). In the end, I opted to keep the currentTimeRange and the priorYearTimeRange as that seemed a thematically consistent pattern for getting the time frames. I am wondering if we should add start and end time for the compare to getInputsForRetrieval, or maybe split the 'time' selectors off into their own. I like the one-stop-shopping convenience of the getInputsForRetrieval, but don't want to overload its function.

Reply:
I'd have to think about that. getInputsForRetrieval was meant to essentially be a short cut for getting the inputs for retrieveHydrographData which gets called in multiple places so adding to it doesn't seem right to me. I'm fine with leaving it as you wrote it.